python/social_bot/envs/play_ground.py

```diff
@@ -104,6 +107,9 @@ def __init__(self,
                 environment step. for the tasks need higher control frequency (such as the 
                 tasks need walking by 2 legs), using a smaller step_time like 0.05 is better.
                 experiments show that iCub can not learn how to walk in a 0.1 step_time
+            real_time_update_rate (int): max update_rate per seconds. there is no limit if
+                this is set to 0. if 1:1 real time is prefered(like playing or recording video),
+                this should be set to 1.0/world_time_precision.
```

Comment:
If the recommended value is 1.0/world_time_precision, then maybe default to None, if None set as 1.0/world_time_precision, otherwise, set to specified value?

Reply:
It's an option in world file, by default it's 0 which means unlimited update rate, i.e., run as fast as possible. In most case (training or evaluating) we'll want this. 
1:1 real time is only sued for playing-back or recording video which we can set it manually if needed.